### ISSUE TITLE
Error cleaning cache

### ISSUE THREAD

**Issue by gpsietzema** [modxcms/pThumb#10]
I can't clear my cache anymore. I get a clean console with just this text: "Console running..."

My apache error log states:
[Tue Nov 12 19:08:45 2013] [error] [client xx.xx.xx.xx] PHP Parse error:  syntax error, unexpected ':' in /core/components/phpthumbof/model/pthumbcachecleaner.class.php on line 100, referer: http://www.xxx.tld/manager/? 

// fixed it, will hand in pull request

**Comment by oo12** [comment]
Ah, you must be using PHP < 5.3?  I didn't realize that ternary operator shortcut `$a ? : $b` was only introduced in 5.3.  Would you try [this version](https://raw.github.com/oo12/phpThumbOf/pThumb-dev/core/components/phpthumbof/model/pthumbcachecleaner.class.php) instead?  Copy and paste it into core/components/phpthumbof/model/pthumbcachecleaner.class.php.

Sorry about that!

**Comment by gpsietzema** [comment]
Yes, PHP 5.2.17. Your version above works by the way. Changing the ternary from '? :'  to '==' works too btw.

**Comment by oo12** [comment]
It's not quite the same though, is it?

```
 $test = 42;

 $shortcut = $test ? : 'no';  // 42: new 'shortcut' syntax with middle value left out (PHP >= 5.3)
 $verbose = $test ? $test : 'no';  // 42: shortcut expands to this expression
 $comparison = $test == 'no'; // false: syntax ok, but different result
```

Thanks for pointing this problem out though.  I'll put the fixed version on Extras tonight!

**Comment by oo12** [comment]
Fixed in 2.2.1.
Thanks again for taking the time to report this.
